docs/en/api/metrics-query-expression.md

```diff
@@ -293,6 +293,21 @@ The following example would return the content of the **service_cpm** metric.
 view_as_seq(not_existing, service_cpm)
 ```
 
+### IsPresent Operation
+IsPresent operation represents that in a list of metrics, if any expression has a value, it would return `1` in the result; otherwise, it would return `0`.
+
+Expression:
+```text
+is_present([<expression_1>, <expression_2>, ...])
+```
+
+For example:
+When the meter does not exist or the metrics has no value, it would return `0`. 
+However, if the metrics list contains meter with values, it would return `1`.
+```text
+is_present(not_existing, existing_without_value, exiting_with_value)
+```
+
 #### Result Type
 The result type is determined by the type of selected not-null metric expression.
```

Comment:
The result type should be `SINGLE_VALUE`, not the same as the` view_as_seq` result.

Reply:
Updated, please re-check